### PR TITLE
[Homepage] Big Teaser cards block

### LINKED ISSUE #13: [Homepage] Implement "big teaser" card block
- [x] implement as option of cards block
- [x] support for background image
- [x] support for headline incl. icon with different font size based screen width
- [x] support for 1-3 content rows on hover (desktop) / below (mobile)

Background color of the area this is used is via #4 
Screenshot 2022-12-15 at 10 54 26

### PR DESCRIPTION
**Pull request by asthabh23**
Fix #13 

Test URLs:
- Before: https://main--eder-group--hlxsites.hlx.page/
- After: https://issue-13--eder-group--hlxsites.hlx.page/

### CODE REVIEW

File: blocks/cards/cards.css [hunk 1]
```diff
@@ -24,58 +24,217 @@
...
 
-.background-mid-grey .default-content-wrapper {
+.cards-container .default-content-wrapper {
     text-align: center;
 }
 
-.background-mid-grey .default-content-wrapper h2 {
+.cards-container .default-content-wrapper h2 {
     font-weight: bold;
     color: white;
```
Comment (mhaack) [inline blocks/cards/cards.css:38]: lets use color variables here

File: blocks/cards/cards.css [hunk 2]
```diff
@@ -24,58 +24,217 @@
...
+    bottom: 0;
+    transform: translate(-50%, -50%);
 }
 
-.cards.mini-teaser .cards-card-body p {
+.cards-wrapper .teaser .cards-card-body h4 {
     font-size: 14px;
+    font-weight: normal;
+    text-align: center;
+    color: black;
```
Comment (mhaack) [inline blocks/cards/cards.css:92]: lets use color variables here

File: blocks/cards/cards.css [hunk 3]
```diff
@@ -24,58 +24,217 @@
...
 
-.cards.mini-teaser .cards-card-body .button-container a {
+.cards-wrapper .teaser .card-text-wrapper .cards-card-body:nth-child(n) {
+    background-color: white;
+}
+
+.cards-wrapper .teaser .cards-card-body h5:last-child a {
     background: none;
     border: none;
     color: #de010a;
```
Comment (mhaack) [inline blocks/cards/cards.css:104]: lets use color variables here

File: blocks/cards/cards.css [hunk 4]
```diff
@@ -24,58 +24,217 @@
...
+
+.cards-wrapper .teaser .cards-card-body h5:last-child {
+    display: inline-block;
+    text-align: center;
+    margin-left: 50px;
+    margin-bottom: 20px;
+}
+
+.cards-wrapper .teaser .card-text-wrapper .cards-card-body:nth-child(2n) {
+    background-color: lightgray;
```
Comment (mhaack) [inline blocks/cards/cards.css:161]: lets use color variables here

File: blocks/cards/cards.css [hunk 5]
```diff
@@ -24,58 +24,217 @@
...
+.cards-wrapper .teaser ul {
+    padding-left: 0;
+} 
+
+.cards-wrapper .teaser ul li {
+    margin-bottom: 20px;
+} 
+
+.cards-wrapper .icon {
+    color: white;
```
Comment (mhaack) [inline blocks/cards/cards.css:50]: lets use color variables here

File: blocks/cards/cards.css [hunk 6]
```diff
@@ -24,58 +24,217 @@
...
+    margin-bottom: 0;
+}
+
+.cards-wrapper .teaser .card-with-image img {
+    display: block;
+    width: 100vw;
+}
+
+.cards-wrapper .teaser > ul > li > .card-with-image h3 {
+    color: white;
```
Comment (mhaack) [inline blocks/cards/cards.css:77]: lets use color variables here

File: blocks/cards/cards.css [hunk 7]
```diff
@@ -24,58 +24,217 @@
...
+    font-weight: normal;
+    text-align: center;
+    color: black;
+    margin-top: 5px;
+    margin-bottom: 0;
 }
 
-.cards.mini-teaser .cards-card-body .button-container a {
+.cards-wrapper .teaser .card-text-wrapper .cards-card-body:nth-child(n) {
+    background-color: white;
```
Comment (mhaack) [inline blocks/cards/cards.css:98]: lets use color variables here

File: blocks/cards/cards.css [hunk 8]
```diff
@@ -24,58 +24,217 @@
...
 .cards.mini-teaser .cards-card-body .button-container a:hover {
     text-decoration: underline;
 }
 
+.cards-wrapper .teaser .cards-card-body h5:last-child:hover a {
+    text-decoration: none;
+}
+
+.cards-wrapper .teaser .card-text-wrapper .cards-card-body h5:nth-last-child(2) a {
+    color: black;
```
Comment (mhaack) [inline blocks/cards/cards.css:122]: lets use color variables here

File: blocks/cards/cards.css (hunk 1 above)
Comment (asthabh23) [reply blocks/cards/cards.css]: fixed in [08e8455](https://github.com/hlxsites/eder-group/pull/79/commits/08e8455b24f90fec35b5c19aeec2d8c92b67c3e6)

File: blocks/cards/cards.css (hunk 2 above)
Comment (asthabh23) [reply blocks/cards/cards.css:92]: fixed in [08e8455](https://github.com/hlxsites/eder-group/pull/79/commits/08e8455b24f90fec35b5c19aeec2d8c92b67c3e6)

File: blocks/cards/cards.css (hunk 3 above)
Comment (asthabh23) [reply blocks/cards/cards.css]: fixed in [08e8455](https://github.com/hlxsites/eder-group/pull/79/commits/08e8455b24f90fec35b5c19aeec2d8c92b67c3e6)

File: blocks/cards/cards.css (hunk 4 above)
Comment (asthabh23) [reply blocks/cards/cards.css]: fixed in [08e8455](https://github.com/hlxsites/eder-group/pull/79/commits/08e8455b24f90fec35b5c19aeec2d8c92b67c3e6)

File: blocks/cards/cards.css (hunk 5 above)
Comment (asthabh23) [reply blocks/cards/cards.css]: fixed in [08e8455](https://github.com/hlxsites/eder-group/pull/79/commits/08e8455b24f90fec35b5c19aeec2d8c92b67c3e6)

File: blocks/cards/cards.css (hunk 6 above)
Comment (asthabh23) [reply blocks/cards/cards.css]: fixed in [08e8455](https://github.com/hlxsites/eder-group/pull/79/commits/08e8455b24f90fec35b5c19aeec2d8c92b67c3e6)

File: blocks/cards/cards.css (hunk 7 above)
Comment (asthabh23) [reply blocks/cards/cards.css]: fixed in [08e8455](https://github.com/hlxsites/eder-group/pull/79/commits/08e8455b24f90fec35b5c19aeec2d8c92b67c3e6)

File: blocks/cards/cards.css (hunk 8 above)
Comment (asthabh23) [reply blocks/cards/cards.css]: fixed in [08e8455](https://github.com/hlxsites/eder-group/pull/79/commits/08e8455b24f90fec35b5c19aeec2d8c92b67c3e6)

**Comment by mhaack** [comment]
Original:
Screenshot 2023-01-18 at 16 54 31

Franklin:
Screenshot 2023-01-18 at 16 54 40

Some minor UI bugs:
* Text should left instead of center
* "weitere Infos" and "Zur Webseite" are not aligned
* Row height is to big

**Comment by mtilburgadobe** [comment]
looks good, and also looks like all the comments from Markus were addressed... approving...

**Review by mtilburgadobe — APPROVED** [review]

**Review by mhaack — APPROVED** [review]

**Pull request merged**